client/verta/verta/client.py

```diff
@@ -660,27 +662,29 @@ def find_datasets(self,
         list of :class:`Dataset`
 
         """
+        datasets = Datasets(self._conn, self._conf)
+        if dataset_ids:
+            datasets._msg.dataset_ids.extend(_utils.as_list_of_str(dataset_ids))
```

Comment:
I'd recommend adding a public interface to avoid cross-object coupling of internals.